app/root.tsx

```diff
@@ -58,13 +58,17 @@ export async function loader({context}: LoaderFunctionArgs) {
     shopId: layout.shop.id,
   };
 
-  return {
+  return json({
     siteSettings,
     layout,
     isPreviewModeEnabled,
     analytics,
     customizerMeta: pack.preview?.session.get('customizerMeta'),
-  };
+  }, {
```

Comment:
Can we do this higher up so devs don't need to do this themselves? If the `session-id` is used for analytics for billing, they can get around billing by not committing the session id